library/src/main/java/com/novoda/downloadmanager/LiteDownloadFileId.java

```diff
@@ -0,0 +1,42 @@
+package com.novoda.downloadmanager;
+
+final class LiteDownloadFileId implements DownloadFileId {
+
+    private final int id;
+
+    LiteDownloadFileId(int id) {
```

Comment:
No more static constructors 🎉  everything goes through the `DownloadFileIdCreator` which returns only the publicly accessible interface.